src/main/java/uk/gov/digital/ho/hocs/casework/util/SnsStringMessageAttributeValue.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.digital.ho.hocs.casework.util;
```

Comment:
How do we do this elsewhere? is this really a problem?

Reply:
'hocs-case-creator' has this class, there was a comment on the remove camel PR to make it a class to simplify the instantiating.